travis/shared_configs/doctr-upload.yml

```diff
@@ -0,0 +1,39 @@
+jobs:
+  include:
+    - stage: deploy
+      name: "Docs Upload"
+      install:
+
+        - echo "Req File':' ${DOCS_REQUIREMENTS:=docs-requirements.txt}"
+        - echo "Folder':' ${DOCS_FOLDER:=docs}"
+        - |
+          # Install docs requirements
+          if [[ -f "${DOCS_REQUIREMENTS}" ]]; then
+            pip install -Ur ${DOCS_REQUIREMENTS}
+          fi
+      before_script:
+        - |
+          # Create HTML documentation
+          set -e
+          pushd ${DOCS_FOLDER}
+          make html
+          popd
+        - |
+          if [[ ! -z "$NO_JEKYLL" ]]; then
+              NO_JEKYLL_COMMAND='--command "touch .nojekyll; git add .nojekyll"'
+          fi
+        - |
+          if [[ ! -z "$DOCTR_VERSIONS_MENU" ]]; then
+              DOCTR_VERSIONS_COMMAND='--command "doctr-versions-menu"'
```

Comment:
You rightfully noted above that both of these shouldn't be set, but if they are mistakenly the command below will expand to:

```bash
 doctr deploy --build-tags $DEPLOY_DIR --deploy-branch-name gh-pages
--command "touch .nojekyll; git add .nojekyll" --command "doctr-versions-menu"
```

Perhaps it should then check that both aren't set, or alternatively reorder the commands so `--command` can never be passed to `git`.

Reply:
Would adding a semicolon to the end of `git add .nojekyll` fix the issue?

Somewhat related issue: the build is currently failing when using the no jekyll command. It seems to be ignoring the quotes.
https://travis-ci.com/github/hhslepicka/testproject1234/jobs/332934809
I tried playing around with different levels of using quotes but couldn't get it to work. Any advice?

Reply:
Would adding something like https://www.sphinx-doc.org/en/master/usage/extensions/githubpages.html help to remove the nojekyll command?

This command also would only affect newly created repos and usually once we create the gh-pages we commit the `.nojekyll` file.

Reply:
Yes, that would be helpful. Much simpler to just do away with the command. Thanks for finding it.